src/components/Personal-Zone/Searcher/AddResumeDialog/ProgressBar/Steps/PreviousJobs.tsx

```diff
@@ -186,7 +187,6 @@ const PreviousJobsStep: React.FC<PreviousJobsStepProps> = (props): JSX.Element =
                     disabled={previousJobs.length === 3}>
```

Comment:
Hi, I know it's not your code.

But if you could add: `const MAX_NUM_PREVIOUS_JOBS_EXPERIENCE=3` (in top of file, for example), and use it both here and in component above when `previousJobs.length !==3`, it would be cool 👍 😃

Reply:
sure